sources/util.go

```diff
@@ -76,7 +76,16 @@ func (a *ARN) ResourceID() string {
 	})
 
 	// Remove the first field since this is the type, then keep the rest
-	return a.Resource[separatorLocation+1:]
+	f := a.Resource[separatorLocation+1:]
+
+	// Check if there is a forward slash in the resource ID
+	// If yes, return the last part of the string
+	s := strings.Split(f, "/")
+	if len(s) > 1 {
+		return s[len(s)-1]
+	}
+
+	return f
```

Comment:
This would pretty substantially change the behaviour right? What's the example where we would want it to do this? I want to be 100% sure we're doing the right thing

Reply:
https://github.com/overmindtech/aws-source/pull/608#discussion_r1751010177

Reply:
The missing account id might be the reason for this behaviour.
See: https://github.com/overmindtech/aws-source/pull/608#discussion_r1752918046